catalog.tests.bom

```diff
@@ -87,7 +87,7 @@ brooklyn.catalog:
           - equals: true
         - type: org.apache.brooklyn.test.framework.SimpleShellCommandTest
```

Comment:
FYI we should be using `TestSshCommand`. But that's separate from this PR, so don't worry about it here. We'll likely address lots of that stuff when we make a concerted effort to remove duplication from lots of these repos (e.g. to rely on https://github.com/brooklyncentral/common-catalog-utils/blob/master/common-tests/src/main/resources/commontests/common.tests.bom).